vlog4j-core/src/test/java/org/semanticweb/vlog4j/core/vlog/VLogExpressions.java

```diff
@@ -0,0 +1,52 @@
+package org.semanticweb.vlog4j.core.vlog;
+
+/*-
+ * #%L
+ * VLog4j Core Components
+ * %%
+ * Copyright (C) 2018 VLog4j Developers
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+/**
+ * Utility class with static methods for creating {@code karmaresearch.vlog}
+ * objects.
+ * 
+ * @author Irina.Dragoste
+ *
+ */
+public final class VLogExpressions {
```

Comment:
Again, I would make this package private. Users should not create such objects.

Reply:
done